androidApp/src/main/java/com/mbta/tid/mbta_app/android/component/UpcomingTripView.kt

```diff
@@ -60,57 +60,65 @@ fun formatTime(time: Instant): String =
     format.format(time.toLocalDateTime(TimeZone.currentSystemDefault()).toJavaLocalDateTime())
 
 @Composable
-fun UpcomingTripView(state: UpcomingTripViewState) {
+fun UpcomingTripView(state: UpcomingTripViewState, hideRealtimeIndicators: Boolean = false) {
```

Comment:
suggestion: maybe worth a test that the indicator image is included when expected